pep-0723.rst

```diff
@@ -0,0 +1,185 @@
+PEP: 723
+Title: Embedding metadata in single-file scripts
+Author: Ofek Lev <ofekmeister@gmail.com>
+PEP-Delegate: Brett Cannon <brett@python.org>
+Discussions-To: https://discuss.python.org/t/29905
+Status: Draft
+Type: Standards Track
+Topic: Packaging
+Content-Type: text/x-rst
+Created: 04-Aug-2023
+Post-History: `19-Jul-2023 <https://discuss.python.org/t/29905>`__
+
+
+Abstract
+========
+
+This PEP specifies a format for defining metadata about single-file Python
+scripts, such as their runtime dependencies. This PEP merely extends
+:pep:`722` with an alternative format and therefore elides describing the
+motivation for such a proposal.
```

Comment:
I don't think this is reasonable - the motivation for PEP 722 leads directly into a number of the points of contention in the discussions. You're welcome to copy the "Motivation" and "Rationale" sections from PEP 722 exactly as written (although I doubt the Rationale would work), but if you don't, please include your own here.